test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -137,6 +148,151 @@ public Object invoke(Object proxy, Method method, Object[] args)
   }
 
   @Test
+  public void testRowToText_specialCharacters() throws Exception {
+    executeUpdate(
+        "create table data (id integer, name varchar, quote varchar)");
+    String sql = "insert into data (id, name, quote) values (1, ?, ?)";
+    try (PreparedStatement ps = getConnection().prepareStatement(sql)) {
+        ps.setString(1, "Rhett Butler");
+        ps.setString(2, "\"Frankly Scarlett, I don't give a damn!\"");
+      assertEquals(1, ps.executeUpdate());
+    }
+
+    MockResponse bar = new MockResponse();
+    Response response = newProxyInstance(Response.class, bar);
+    Map<String, String> cfg = new TreeMap<String, String>();
+    ResponseGenerator resgen = ResponseGenerator.rowToText(cfg);
+
+    ResultSet rs = executeQueryAndNext("select * from data");
+    resgen.generateResponse(rs, response);
+    String golden = "DATA,DATA,DATA\nID,NAME,QUOTE\n"
+        + "1,Rhett Butler,\"\"\"Frankly Scarlett, I don't give a damn!\"\"\"\n";
+    assertEquals(golden, bar.baos.toString(UTF_8.name()));
+  }
+
+  @Test
+  public void testContentColumnModeServesResultVarchar() throws Exception {
```

Comment:
What about dropping the verbosity of these names: testContentColumn_varchar, etc?